### DISCUSSION TITLE
关于智能插座的电量统计功能

### DISCUSSION THREAD

**Discussion by 799621482** [XiaoMi/ha_xiaomi_home#77] [Ideas]
希望可以给智能插座加入日、月耗电量统计

**Comment by lzcapp** [comment]
➕1️⃣
![image](https://github.com/user-attachments/assets/3d6aedfe-e35e-4cd7-85d9-5dbe840a6b94)

**Comment by JimmyKmi** [comment]
该设备没有将此参数作为spec上报，故无法获取
具体spec列表参考 https://mijia.wiki/products
